utils/merge_and_push_translations.sh

```diff
@@ -54,89 +54,141 @@ fi
 
 echo "Working tree is clean, continuing"
 
-# Print all commands.
-set -x
+update_authors() {
```

Comment:
The source files for update_authors.py are data/i18n/locales AND data/txts/developers.json

The 2nd is not changed by transifex, so your test is fine for the 1st ("translation") commit.
But the script has to run (before the 2nd) commit in case a new developer was added (last time in 13632b60). It takes less than a second to run it, so this could run for the 2nd commit anyway. (see `d)` in [my comment](#issuecomment-1685073340))

--
Ah, data/ is missing in the paths in the comments in `update_authors.py`, I can update this soon.

Reply:
Actually I'm OK without these checks (if potential sources are scattered, then it's even better to not do them – that's part of what I meant by trusting git), I just wanted to show what I meant in earlier comments.